openwisp_utils/admin_theme/context_processor.py

```diff
@@ -3,6 +3,11 @@
 from django.contrib.admin import site
 from django.urls import reverse
 
+try:
+    from openwisp_utils.admin_theme.site import admin_site
+except Exception:  # pragma: no cover
```

Comment:
I don't understand why are you doing this. Can you explain? It looks like something that shouldn't be done.
Why can it fail and why should it fail silently instead of loudly?

Reply:
Oh, you are right there is no need for that, there is no way that it can fail